src/store/database/actions.js

```diff
@@ -1,6 +1,10 @@
-const writeToDatabase = data => ({
+const writeToDatabase = (movies, lists, genres) => ({
```

Comment:
Why do you need these params? Just pass object `entities` after normalization, and you will have these values like:
```
{
  movies: {...},
  lists: {...},
  genres: {...},
}
```